mjk-examples/OCF/odmthing-ocf-washer-new.sdf.json

```diff
@@ -15,22 +15,22 @@
     "oic.d.washer" : {
       "name" : "oic.d.washer",
       "title" : "washing machine",
-      "required": [
-        "0/odmObject/powerswitch",
-        "0/odmObject/washerstate"
+      "odmRequired": [
+        "#/odmThing/oic.d.washer/odmObject/powerswitch",
+        "#/odmThing/oic.d.washer/odmObject/washerstate"
       ],
       "odmObject": {
         "powerswitch": {
-          "odmRef": "<library>#/odmObject/oic.r.switch.binary"
+          "odmRef": "ocf:/#odmObject/oic.r.switch.binary"
         },
         "washerstate": {
-            "odmRef": "<library>#/odmObject/oic.r.operational.state"
+            "odmRef": "ocf:/#odmObject/oic.r.operational.state"
         },
         "washermode": {
-            "odmRef": "<library>#/odmObject/oic.r.mode"
+            "odmRef": "ocf:/#odmObject/oic.r.mode"
```

Comment:
same here, not sure if this syntax is supported

Reply:
I just removed this example because we don't have namespace references working yet.